app.py

```diff
@@ -18,10 +18,27 @@
 def homepage():
     return render_template('index.html')
 
+    """ This renders the homepage
```

Comment:
IDE doesn't like this because the docstring is outside of the function. Consider moving this docstring inside of the function definition (in between 18 and 19) avoid that. Code works fine either way on my end.